### ISSUE TITLE
Image Similarity Search Code Example

### ISSUE THREAD

**Issue by Ferev** [keras-team/keras-io#307]
Hello,
I am working on ["Image Similarity Search" ](https://github.com/keras-team/keras-io/blob/master/call_for_contributions.md#image-similarity-search-engine-using-triplet-loss) with a team of other students.
Please let me know if there are any particular requirements or people who are currently working on it.
Thanks!

**Comment by fchollet** [comment]
Sounds good! Let me know if you have any questions about the example (better to email me directly). As far as I know no one else is working on it at this time.

**Comment by sachinprasadhs** [comment]
Hi @Ferev,

It looks like the example `Image similarity using Siamese Network with a triplet loss` is updated here https://keras.io/examples/vision/siamese_network/ .

If this is what you are looking for, please spare some time to close the issue. Thanks!

**Comment by Ferev** [comment]
Thanks Sachin, closing this.
